hudi-hadoop-common/src/main/java/org/apache/hudi/hadoop/fs/HadoopFSUtils.java

```diff
@@ -196,6 +215,10 @@ public static FileStatus convertToHadoopFileStatus(StoragePathInfo pathInfo) {
         convertToHadoopPath(pathInfo.getPath()));
   }
 
+  public static void setConfForConfigurableInputFormat(FileInputFormat inputFormat, Configuration conf) {
```

Comment:
Some of these are only used by test classes.  If so, could you move them to test util classes, so production code should not use them?